featurebyte/core/timedelta.py

```diff
@@ -35,10 +35,19 @@ def to_timedelta(series: Series, unit: TimedeltaSupportedUnitType) -> Series:
 
     Examples
     --------
+    Create a new column in the INVOICEITEMS view that represents the number of days since the last invoice.
+
     >>> items_view = catalog.get_view("INVOICEITEMS")
-    >>> items_view["DAYS_SINCE_LAST_INVOICE"] = to_timedelta(  # doctest: +SKIP
+    >>> items_view["TIMEDELTA_DAYS_SINCE_LAST_INVOICE"] = fb.to_timedelta(  # doctest: +SKIP
     ...   items_view["DAYS_SINCE_LAST_INVOICE"], "days"
     ... )
+
+    Create a new column that is 10 minutes after the event.
+
+    >>> items_view = catalog.get_view("INVOICEITEMS")
+    >>> items_view["10_MINS_AFTER_EVENT"] = fb.to_timedelta(  # doctest: +SKIP
+    ...   items_view["EVENT_TIMESTAMP"], "minutes"
+    ... ) + 10
```

Comment:
I don't think this is the correct syntax since the first argument for `fb.to_timedelta` should be an integer column / feature. Since 10 mins is a constant timedelta, `pd.Timedelta` should be used in that case:

```
items_view["10_MINS_AFTER_EVENT"] = items_view["EVENT_TIMESTAMP"] + pd.Timedelta(10, unit="minute")
```

Reply:
Would be good not to skip the doctest and these statements should not have side effect on the original table.